src/components/atoms/Callout/Callout.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './Callout.scss'
+
+const Callout = ({ className, style, content }) => (
+  <article className={`cg-callout ${className}`} style={{ ...style }}>
+    <div>{content}</div>
```

Comment:
Remove the empty div from here. this div is not required.

Reply:
Also, i would say instead of content props you can use the child component way. this way we will have more flexibility of rendering anything.

like:
<article>{children}</article>

while using <Callout>"Some Text" </Callout>